### PR TITLE
Add a document on installing the VMware VDDK on containers

### PR DESCRIPTION
**Pull request by bdunne**
Based on:
- https://github.com/ManageIQ/manageiq-pods/issues/538
- https://github.com/ManageIQ/manageiq-pods/pull/666
- https://github.com/ManageIQ/manageiq-pods/issues/635

Fixes: https://github.com/ManageIQ/manageiq-documentation/issues/1479

### CODE REVIEW

File: installing_on_kubernetes/_topics/installing_vddk.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        concerning backward and forward compatibility for VDDK.
+
+2. Set up a temporary directory where we will build the image
+```bash
+$ mkdir -p /tmp/vddk_container/container-assets
+```
+
+3. Copy the tar file into the container-assets directory.  For Example:
+```bash
+$ cp ~/Downloads/VMware-vix-disklib-7.0.1-16860560.x86_64.tar.gz /tmp/vddk_container/container-assets
```
Comment (jrafanie) [inline installing_on_kubernetes/_topics/installing_vddk.md:27]: Should this be a generic version value and we point to another document describing what versions of vddk we support and against which versions of vsphere?  cc @agrare

File: installing_on_kubernetes/_topics/installing_vddk.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+RUN cd /vddk && \
+    filename=$(find . -name VMware-*) && \
+    if [[ -f $filename ]] ; then tar -zxvf VMware-* ; fi && \
+    mkdir -p vmware-vix-disklib-distrib && \
+    touch /vddk/vmware-vix-disklib-distrib/.keep
+
+################################################################################
+
+### IMPORTANT: Modify the following image and tag as necessary to reflect the version that you're running
+FROM docker.io/manageiq/manageiq-base-worker:latest
```
Comment (jrafanie) [inline installing_on_kubernetes/_topics/installing_vddk.md:45]: Should this  point to a non-existing image and tag so they can't forget to update this value?

File: installing_on_kubernetes/_topics/installing_vddk.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+COPY --from=vddk /vddk/vmware-vix-disklib-distrib/ /usr/lib/vmware-vix-disklib/
+
+RUN /tmp/install-vmware-vddk
+```
+
+5. Build the image and tag it appropriately for your registry, then push it to the registry.
+```bash
+$ cd /tmp/vddk_container/
+$ docker build . -t your_registry.example.com/namespace/manageiq-base-worker:latest_vddk
+$ docker push your_registry.example.com/namespace/manageiq-base-worker:latest_vddk
```
Comment (jrafanie) [inline installing_on_kubernetes/_topics/installing_vddk.md:56]: I like this example registry, image and tag.  They can't forget to update it and the it's very clear what needs to be done here.

File: installing_on_kubernetes/_topics/installing_vddk.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+```
+
+6. Update the CR to use the base worker image you just pushed to the registry.
+```yaml
+baseWorkerImage: your_registry.example.com/namespace/manageiq-base-worker:latest_vddk
+```
+
+The operator will now update the orchestrator and worker deployments to reflect this change.
```
Comment (jrafanie) [inline installing_on_kubernetes/_topics/installing_vddk.md:64]: What do they need to do if we release a new image?  Ideally, it's just changing the 
`FROM ...` and updating the revision of their image tag they build and push, right?

Comment (bdunne) [reply installing_on_kubernetes/_topics/installing_vddk.md:64]: Yes, they will need to rebuild anytime we update the image.

File: installing_on_kubernetes/_topics/installing_vddk.md (hunk 1 above)
Comment (bdunne) [reply installing_on_kubernetes/_topics/installing_vddk.md]: Done

File: installing_on_kubernetes/_topics/installing_vddk.md (hunk 2 above)
Comment (bdunne) [reply installing_on_kubernetes/_topics/installing_vddk.md]: Done

**Review by jrafanie — APPROVED** [review]
LGTM

**Pull request merged**